code/modules/mining/equipment_locker.dm

```diff
@@ -507,8 +507,8 @@
 	if(istype(proj_turf, /turf/simulated/mineral))
 		var/turf/simulated/mineral/M = proj_turf
 		playsound(src,'sound/weapons/resonator_blast.ogg',50,1)
-		M.gets_drilled()
 		spawn(5)
+			M.gets_drilled(creator)
```

Comment:
Why'd you move this to be inside the spawn()? Won't this mean that the explosion will only trigger 5ds after shooting?